CRM/Contact/BAO/Group.php

```diff
@@ -900,115 +915,108 @@ public static function getGroupList(&$params) {
     $visibility = CRM_Core_SelectValues::ufVisibility();
 
     while ($object->fetch()) {
-      $permission = CRM_Contact_BAO_Group::checkPermission($object->id, TRUE);
```

Comment:
Per comments I removed the $permission as the code above does already limit the groups based on ACLs & this introduces some caching weirdness when disabled groups are involved. Rest of change is just whitespace.

Tests in CRM_Group_Page_AjaxTest are v thorough